File: April 17, 2026.py
def decode(message):

    key = "VLHCGMDLNH"
    key_i = 0
    decoded_message = ""

    for char in message: 
        if char.isalpha():
            key_char = key[key_i % len(key)]
            key_index = ord(key_char.upper()) - ord('A') + 1
            char_index = ord(char.upper()) - ord('A') + 1
            new_index = (char_index - key_index - 1) % 26 + 1
            decoded_message += chr(new_index + ord('A') - 1)
            key_i += 1
        else:
            decoded_message += char

    return decoded_message

decode("YAVJYNXE")

File: April 17, 2026.js
function decode(message) {

  const key = "VLHCGMDLNH";
  let keyI = 0;
  let decodedMessage = "";

  for (let char of message) {
    if (char.toLowerCase() !== char.toUpperCase()) {
      let keyChar = key[keyI % key.length];
      let keyIndex = keyChar.toUpperCase().charCodeAt(0) - "A".charCodeAt(0) + 1;
      let charIndex = char.toUpperCase().charCodeAt(0) - "A".charCodeAt(0) + 1;
      let newIndex = (charIndex - keyIndex - 1 + 26) % 26 + 1;
      decodedMessage += String.fromCharCode(newIndex + "A".charCodeAt(0) - 1 );
      keyI += 1;
    } else {
      decodedMessage += char;
    }
  }

  return decodedMessage;
}